src/Spec/Swagger2.php

```diff
@@ -125,6 +125,156 @@ public function getServices()
         return $list;
     }
 
+    protected function convertMethod(string $methodName, string $pathName, array $method): array
```

Comment:
Let's make the name a bit more descriptive, I can't tell what it does from the name